domains/yunexiz.json

```diff
@@ -1,7 +1,8 @@
 {
   "owner": {
-    "username": "Yunexiz",
-    "email": "yunexiz@aol.com"
+    "discord_username": "yunexiz",
+    "discord_id": "1040303561847881729",
+    "email": "141266522+Yunexiz@users.noreply.github.com"
```

Comment:
I'm not sure if I can allow this, since in a docs about PRs this isnt really allowed, but you could just leave the email line blank since you have the discord line

Reply:
done